### DISCUSSION TITLE
Adding monitor interface

### DISCUSSION THREAD

**Discussion by issmonitor** [Security-Onion-Solutions/securityonion#1585] [Unsupported Versions] [answered]
Does anyone know how to add a monitoring interface on the management server.  I have a physical server that on the previous version was a management server but also had a monitoring interface.  Not sure if I missed it during the install process or not.  

Thanks

**Comment by TOoSmOotH** [comment, marked as answer]
You cannot convert a management server to a sensor. You will need to offer more information on what you are trying to do and how you have your distributed setup configured. Otherwise you would just need to use the standalone option during install.

**Reply by issmonitor** [reply]
in the previous setup, we had the management server was also a sensor and then I also had another forwarding node.  it is sounding like i will need to just setup another forwarding node instead of having the management server unless I can do a standalone install and then attach a forwarding node to it.

**Reply by TOoSmOotH** [reply]
You should be able to use standalone and connect the remote forward node to it. This will give you the functionality you are looking for.

**Reply by issmonitor** [reply]
Thanks
